docs/headers/ti/getcsc.rst

```diff
@@ -13,6 +13,35 @@ You can easily use it instead of the :code:`keypadc` library, and is particularl
 The OS's interrupt handler handles the keyboard driver and caches a scancode when it detects a key press.
 You can use this scancode directly without any translation of 2nd and alpha through the :code:`os_GetCSC` function.
 
+TI-83 Premium CE
+----------------
+
+Some keys on the TI-83 Premium CE are labeled differently, and some have completely different functions.
+For convenience, there are duplicate equates for the five keys with different functions;
+in the table below, they are **emphasized.**
+
++---------------+-------------------+-----------------------+-------------------------+
+| TI-84 Plus CE | Equate            | TI-83 Premium CE      | Equate                  |
++===============+===================+=======================+=========================+
+| Y=            | :code:`sk_Yequ`   | f(x)                  | :code:`sk_Yequ`         |

```

Comment:
Should we also provide `#define`s in French  or at least with whatever label the keypad uses?

Reply:
I actually originally named the keys in French. However, I realized that literally everything else in the SDK is in English, and I figured having just that little bit in French might be even more confusing. Do French programmers care about having a `sk_Fenetre` or even `sk_2nde`?

Reply:
I hadn't realized there were equates in keypadc.h, so I've changed getcsc.h to match it.